pallets/validators-manager/src/tests_voting_deregistration.rs

```diff
@@ -78,10 +87,10 @@ fn setup_ext_builder() -> (TestExternalities, Arc<RwLock<PoolState>>, Arc<RwLock
 
     (ext, pool_state, offchain_state)
 }
-
+/// Setups a voting session to deregister collator 5. Sender is collator 3
 fn setup_voting_session(action_id: &ActionId<AccountId>) {
     let collator_eth_public_key = ecdsa::Public::from_raw(hex!(
-        "02407b0d9f41148bbe3b6c7d4a62585ae66cc32a707441197fa5453abfebd31d57"
+        "0362c0a046dacce86ddd0343c6d3c7c79c2208ba0d9c9cf24a6d046d21d21f90f7"
```

Comment:
Can we turn these keys into constants? Perhaps it would make the code easier to follow, as in we'd know immediately which collator we're dealing with